tutorials/R/Lidar/intro-to-lidar/what-CHM-DSM-DTM/what-CHM-DSM-DTM.md

```diff
@@ -18,26 +18,26 @@ urlTitle: chm-dsm-dtm-gridded-lidar-data
 
 ## Common LiDAR Data Products
 
-* <a href="https://www.neonscience.org/3d/SJER_DTM_3d.html" target="_blank">Digital Terrain Model</a> - This 
-product represents the elevation of the ground.
-* <a href="https://www.neonscience.org/3d/SJER_DSM_3d.html" target="_blank">Digital Surface Model</a> - This 
-represents the elevation of the tallest surfaces at that point. Imagine draping 
+* <a href="https://www.neonscience.org/3d/SJER_DTM_3d.html" target="_blank">Digital Terrain Model</a> - This product represents the elevation of the ground.
+* <a href="https://www.neonscience.org/3d/SJER_DSM_3d.html" target="_blank">Digital Surface Model</a> - This represents the elevation of the tallest surfaces at that point. Imagine draping 
 a sheet over the canopy of a forest, the DEM contours with the heights of the 
 trees where there are trees but the elevation of the ground when there is a 
-clearing in the forest.
+clearing in the forest. 
```

Comment:
Check out these images and consider using a graphic here: 
https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/lidar-derived-products/lidarTree-height.png
https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/lidar-derived-products/DSM-DTM.png